examples/apollo-federation-compatibility/docker-compose.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+  server:
```

Comment:
expected service name should be `products`

see: https://github.com/apollographql/apollo-federation-subgraph-compatibility/issues/336#issuecomment-1409284825